NuGet.config

```diff
@@ -12,6 +12,8 @@
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <!-- Used for the SiteExtension 3.1 bits that are included in the 5.0 build -->
     <add key="dotnet31-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1-transport/nuget/v3/index.json" />
+    <!-- Used for the Rich Navigation indexing task -->
+    <add key="richnav" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
```

Comment:
@jepetty @mmitche could this package be included in dotnet-eng instead❔ Seems odd for us to need a new feed for something referenced from the Arcade SDK.